platform/formats/index.mdx

```diff
@@ -82,3 +82,8 @@ With Tolgee, you're able to import and export .arb files. [Read more](flutter_ar
 ## Ruby on Rails (.yaml)
 Ruby YAML is structured in YAML format with array support specific to Ruby on Rails applications.
 [Read more](ruby_yaml.mdx).
+
+## i18next (.json)
+
+The i18next library uses custom format to represent localizations. With Tolgee, you're able to import and export json files with this format, but not all features are supported.
```

Comment:
In all of the other formats, we specify where it's used we don't scare the users about not supported features on the index page.

Reply:
It should be fixed.